.github/workflows/code-climate.yml

```
@@ -0,0 +1,53 @@
+                  version: 11
+                  php_version: 8.3
+                  php_extensions: xdebug shmop curl pdo pdo_mysql
+                  coverage_clover: coverage.xml
+                  bootstrap: vendor/autoload.php
+                  test_suffix: Test.php
+                  args: --coverage-filter src tests
+              env:
+                  XDEBUG_MODE: coverage
+
+            - name: Fix code coverage paths
+              run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:12: Double quote to prevent globbing and word splitting

(shellcheck)